### PR TITLE
Mifosx 1477:hiding deposit and withdraw buttons based on given permissions.

### PR DESCRIPTION
**Pull request by rajeshreddy-k**

### CODE REVIEW

**Comment by vishwasbabu** [comment]
Excellent job @rajeshreddy-k !

**Comment by rajeshreddy-k** [comment]
Thank you sir.

**Comment by rajeshreddy-k** [comment]
Thank you sir :)

On Thu, Apr 9, 2015 at 12:35 PM, Vishwas Babu A J <notifications@github.com>
wrote:

> Excellent job @rajeshreddy-k <https://github.com/rajeshreddy-k> !
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openMF/community-app/pull/1321#issuecomment-91132086>.
>



-- 
Thank you & regards
Rajesh Reddy k
7795405419

File: app/scripts/controllers/client/ViewClientController.js [hunk 1]
```diff
@@ -48,6 +48,26 @@
                     }
                 });
 
+                scope.clickEvent = function (eventName, accountId, savingProductId) {
```
Comment (vishwasbabu) [inline app/scripts/controllers/client/ViewClientController.js]: would be better to have meaningful names:

Method name might be : navigateToSavingsOrDepositAccount
The last parameter passed is productType (not savingProductId)

File: app/scripts/controllers/client/ViewClientController.js [hunk 2]
```diff
@@ -55,6 +75,40 @@
                     scope.buttons = clientStatus.getStatus(data.status.value);
                 }
 
+                if (clientStatus.statusKnown(data.status.value)) {
+                    scope.buttons1 = clientStatus.getStatus(data.status.value);
```
Comment (vishwasbabu) [inline app/scripts/controllers/client/ViewClientController.js]: this is unnecessary, what does it do?

File: app/scripts/controllers/client/ViewClientController.js [hunk 3]
```diff
@@ -55,6 +75,40 @@
...
+
+                    scope.buttons1 = {
```
Comment (vishwasbabu) [inline app/scripts/controllers/client/ViewClientController.js]: use meaningful names wherever possible (ex: "savingsActionbuttons")

**Pull request merged**